src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsSignalingSpec.groovy

```diff
@@ -810,22 +888,338 @@ class PriceFloorsSignalingSpec extends PriceFloorsBaseSpec {
+
+        and: "PBS should log a errors"
+        def logs = floorsPbsService.getLogsByTime(startTime)
+        def floorsLogs = getLogsByText(logs, ERROR_LOG(bidRequest, message))
+        assert floorsLogs.size() == 1
+
+        and: "Alerts.general metrics should be populated"
+        def metrics = floorsPbsService.sendCollectedMetricsRequest()
+        assert metrics[ALERT_GENERAL] == 1
+    }
+
+    def "PBS should use default floors config when original account config is invalid"() {
+        given: "Bid request with empty floors"
+        def bidRequest = bidRequestWithFloors.tap {
+            ext.prebid.floors = null
+        }
+
+        and: "Account with invalid floors config"
+        def accountConfig = new AccountConfig(auction: new AccountAuctionConfig(priceFloors: new AccountPriceFloorsConfig(enabled: false)))
+        def account = new Account(uuid: bidRequest.accountId, config: accountConfig)
+        accountDao.save(account)
+
+        and: "Set bidder response"
+        def bidResponse = BidResponse.getDefaultBidResponse(bidRequest)
+        bidder.setResponse(bidRequest.id, bidResponse)
+
+        and: "Flush metrics"
+        flushMetrics(floorsPbsService)
+
+        when: "PBS processes auction request"
+        def response = floorsPbsService.sendAuctionRequest(bidRequest)
+
+        and: "PBS floors validation failure should not reject the entire auction"
+        assert !response.seatbid?.isEmpty()
+
+        then: "PBS shouldn't log warning or errors"
+        assert !response.ext?.warnings
+        assert !response.ext?.errors
+
+        and: "Alerts.general metrics shouldn't be populated"
+        def metrics = floorsPbsService.sendCollectedMetricsRequest()
+        assert !metrics[ALERT_GENERAL]
+    }
+
+    def "PBS should emit error in log and response when account have disabled dynamic data config"() {
+        given: "Bid request without floors"
+        def bidRequest = bidRequestWithFloors.tap {
+            ext.prebid.floors = null
+        }
+
+        and: "Account with invalid floors config"
+        def account = getAccountWithEnabledFetch(bidRequest.accountId).tap {
+            config.auction.priceFloors.useDynamicData = false
+        }
+        accountDao.save(account)
+
+        and: "Set bidder response"
+        def bidResponse = BidResponse.getDefaultBidResponse(bidRequest)
+        bidder.setResponse(bidRequest.id, bidResponse)
+
+        and: "PBS fetch rules from floors provider"
+        cacheFloorsProviderRules(bidRequest, floorsPbsService, GENERIC, SUCCESS)
+
+        and: "Flush metrics"
+        flushMetrics(floorsPbsService)
+
+        when: "PBS processes auction request"
+        def response = floorsPbsService.sendAuctionRequest(bidRequest)
+
+        then: "PBS should log a warning"
+        def message = "Using dynamic data is not allowed"
+        assert response.ext?.warnings[PREBID]*.code == [999]
+        assert response.ext?.warnings[PREBID]*.message == [message]
```

Comment:
Can be in line:
`assert response.ext?.warnings[PREBID]*.message == ["Using dynamic data is not allowed"]`